tests/utils/commands.py

```diff
@@ -1,13 +1,15 @@
 import subprocess
 
 
-def capture(command, cwd=None):
+def capture(command, cwd=None, shell=False, command_input=None):
```

Comment:
Is the `shell` argument necessary anymore?

Reply:
No, but it might be in the future, if something requires. Futureproofing and all that, yes?

Reply:
I mean, considering that other arguments aren't there for "futureproofing", I don't think it should be here. Perhaps `**kwargs` could be forwarded for better futureproofing, but that seems out of scope for this PR

Reply:
Fair enough. I'll remove it. I do remember that in the original test for this feature, shell mode was used, and was removed when the original test was removed during the refactor of the tests system.